src/main/java/openblocks/client/renderer/block/BlockWorkingProjectorRenderer.java

```diff
@@ -0,0 +1,135 @@
+package openblocks.client.renderer.block;
+
+import net.minecraft.client.renderer.RenderBlocks;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.util.IIcon;
+import net.minecraft.world.IBlockAccess;
+
+import openblocks.Config;
+import openblocks.common.block.BlockWorkingProjector;
+
+import openmods.renderer.IBlockRenderer;
+
+public class BlockWorkingProjectorRenderer implements IBlockRenderer<BlockWorkingProjector> {
+
+	public static int renderPass = 0;
+
+	private static final float TO_BLOCK_CENTRE = 0.5F;
+	private static final int BRIGHTNESS_LEVEL_MAX = 255;
+	private static final int BRIGHTNESS_LEVEL_MIN = 0;
+	private static final int BRIGHTNESS_LEVEL_DEF = -1;
+
+	private static final float CONE_BEGIN = Config.experimentalFeatures? 0F : 0.25F;
```

Comment:
Definitely. Config values are cheap, no point in clumping (possible) multiple functionalities in one.